app/assets/javascripts/components/DashboardHeader.jsx

```diff
@@ -0,0 +1,5 @@
+class DashboardHeader extends React.Component {
+  render() {
+    return (<h1>Hello {this.props.name}</h1>);
+  }
+}
```

Comment:
nit: newline

Reply:
changed in: https://github.com/calblueprint/sjc/pull/14/commits/185ffe235d12ea28170ab87c0c1430389da5af3e